Handle passing no triggers to Combine and First (#4409)
cocotb/cocotb: docs/source/newsfragments/4409.bugfix.1.rst

> Passing no triggers to :class:`~cocotb.triggers.First` used to hang the simulation indefinitely. Now it raises a :exc:`ValueError` exception.

Author: ```suggestion
Passing no triggers to :class:`~cocotb.triggers.First` previously hung the simulation indefinitely. Now, doing so raises a :exc:`ValueError` exception.
```
